runtime/v2/shim.go

```diff
@@ -117,11 +117,12 @@ func loadShim(ctx context.Context, bundle *Bundle, events *exchange.Exchange, rt
 		}
 	}()
 	s := &shim{
-		client:  client,
-		task:    task.NewTaskClient(client),
-		bundle:  bundle,
-		events:  events,
-		rtTasks: rt,
+		client:      client,
+		task:        shimapi.NewTaskClient(client),
+		portforward: shimapi.NewPortForwardClient(client),
```

Comment:
If the shim could just return the client here, then the port forward client could be completely separated here